docs/mevblocker/concepts/mev-concepts/what-is-frontrunning.md

```diff
@@ -1,66 +1,61 @@
 
 ## Example: Frontrunning in DeFi
+
 A notable example of DeFi frontrunning involves the notorious MEV bot named "jaredfromsubway.eth." In one case, a user, [0x3a7822](https://zeromev.org/block?num=17508508), intended to trade 2 ETH for another token, $BOB.
 
 ### How the frontrunning attack was executed
 
 This trade was noticed by the bot jaredfromsubway.eth, which monitored the mempool for transactions it could take advantage of.
 
-Once the transaction entered the mempool, Jaredfromsubway.eth quickly executed its own transaction to buy \$BOB with ETH, strategically placing it as the first in the block, and knowing that he would get the assets at a discount as the transactions behind will increase the price \$BOB. MEV allows validators to prioritize transactions that are of the most value to them. Consequently, the bot's transaction, being the first one in the block, was confirmed ahead of the user's transaction.
+Once the transaction entered the mempool, jaredfromsubway.eth quickly executed its own transaction to buy \$BOB with ETH, strategically placing it as the first transaction in the block, and knowing that he would get the assets at a discount as the transactions behind will increase the price of \$BOB. MEV allows validators to prioritize transactions that are of the most value to them. Consequently, the bot's transaction, being the first one in the block, was confirmed ahead of the user's transaction.
 
-![Frontrunning](/img/mevblocker/Frontrunning_2.webp)
+![Frontrunning](/img/mevblocker/Frontrunning_2.webp) 
 
 ### The consequences of the frontrunning attack
 
-The frontrunning attack triggered an increase in the price of \$BOB tokens due to the additional demand created before the user's transaction was processed. When the user's original order of 145M \$BOB was executed, it was conducted at this now-inflated price, leading to user 0x3a7822 receiving way fewer \$BOB tokens, hence losing out on the value they could have otherwise captured.
+The frontrunning attack triggered an increase in the price of \$BOB tokens due to the additional demand created before the user's transaction was processed. When the user's original order of 145M \$BOB was executed, it was conducted at this now-inflated price, leading to user 0x3a7822 receiving significantly fewer \$BOB tokens, hence losing out on the value they could have otherwise captured.
 
-Jaredfromsubway.eth completed its MEV attack by selling its \$BOB tokens at an increased price, resulting in a profit, a second move that made this MEV attack a sandwich attack. In this scenario, the price slippage triggered by the frontrunning bot meant that user 0x3a7822 encountered a worse trade-off, ultimately making a loss while the MEV bot profited - an estimated gain of 0.1 ETH.
-
```

Comment:
### An alternative frontrunning scenario

If user, 0x3a7822 identified a liquidation opportunity faster than anyone else and sent its transaction to the mempool to get the discounted assets from the liquidation in the first place, then Jaredfromsubway.eth would have seen this and would have replaced the users transactions with the transaction of its own, but with a higher gas fee, and thus more has an increased likelihood to of being processed faster, and effectively stealing the opportunity of capturing value away from the user.

why are we deleting this section @gogonimago ?

Reply:
I don't think there's a need to provide two different frontrunning scenarios. These are docs, not educational guides and if people want to learn more they can read our articles about them. In fact I thought about removing the three MEV explainers entirely from the docs and just sending people to our articles about them.